var/spack/repos/builtin/packages/py-auxlib/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAuxlib(PythonPackage):
+    """Auxlib is an auxiliary library to the python standard library."""
+
+    homepage = "https://github.com/kalefranz/auxlib"
+    url      = "https://pypi.io/packages/source/a/auxlib/auxlib-0.0.43.tar.gz"
+
+    version('0.0.43', sha256='0f175637e96a090a785767ce28483cf1aeec316a19afce9b2fbd113e1122786a')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-enum34', type=('build', 'run'), when='^python@:3.3')
+    depends_on('py-ordereddict', type=('build', 'run'), when='^python@:2.6')
+    depends_on('py-wheel', type=('build', 'run'))
```

Comment:
This dependency shouldn't be needed unless we are installing from a `.whl` file.

Reply:
It will not build without it. I should have added a comment for that, sorry.
```console
==> Error: Failed to install py-auxlib due to ChildError: ProcessError: Command exited with status 1:
    '/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-broadwell/gcc-9.1.0/python-3.7.4-u4qpwrkkkbdy2sujbdfouxlvwxpyicen/bin/python3.7' '-s' 'setup.py' '--no-user-cfg' 'build'
1 error found in build log:
     2    ==> [2020-02-27-12:42:39.808070, 23670] '/opt/packages/gpjohnsn/spack/opt/spack/linux-centos7-broadwell/gcc-9
          .1.0/python-3.7.4-u4qpwrkkkbdy2sujbdfouxlvwxpyicen/bin/python3.7' '-s' 'setup.py' '--no-user-cfg' 'build'
     3    usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     4       or: setup.py --help [cmd1 cmd2 ...]
     5       or: setup.py --help-commands
     6       or: setup.py cmd --help
     7    
  >> 8    error: invalid command 'bdist_wheel'
```

Reply:
Huh, never seen that one before.